test/corpus/expressions.txt

```diff
@@ -739,8 +747,9 @@ baz = 7 unless qux?
 
 ================================================================================
 multiple assignment
+:language(crystal)
 ================================================================================
-foo, bar = bar, foo
+foo, bar = baz, fuz
```

Comment:
The original form was a syntax error:
```
Error: can't use variable name 'bar' inside assignment to variable 'bar'
```

Reply:
it works if you have `foo` and `bar` initialized to something before switching them. https://carc.in/#/r/hkt8

Reply:
Yes, but I didn't want to add a setup phase to the test :ghost: